drivers/storage/scaleio/tests/test-env-up.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+
+# This script launches infrastructure required for testing the ScaleIO storage driver.
+# It spins up VPC and EC2 instance so AWS account owner will get charged for time
+# that resources will be running.
+
+set -e
+# set -x
+
+AWS_ACCESS_KEY="$1"
```

Comment:
Hi @dvonthenen,

Let's give users the ability to set the credentials via environment variables to prevent sensitive information from appearing in process lists or test logs. For example, we could rewrite lines 10-13 like so:

```bash
AWS_ACCESSKEY=${AWS_ACCESSKEY:-$1}
AWS_SECRETKEY=${AWS_SECRETKEY:-$2}
CF_STACK_NAME=${CF_STACK_NAME:-$3}
LAUNCH_KEY_NAME=${LAUNCH_KEY_NAME:-$4}
```

Note that I also changed the names of the AWS credential variables to remove the second underscore in each. I did this to make them consistent with the same configuration variables for our AWS-related storage drivers.